src/main/java/org/identityconnectors/ldap/LdapEntry.java

```diff
@@ -125,31 +128,14 @@ public Attributes getAttributes() {
         public LdapName getDN() {
             if (dn == null) {
                 if (result.isRelative()) {
-                    try {
-                        //dn = join(result.getName(), baseDN);
-                        dn = new LdapName(result.getNameInNamespace());
-                    } catch (InvalidNameException ex) {
-                         throw new ConnectorException(ex);
-                    }
+                    dn = quietCreateLdapName(result.getNameInNamespace());
```

Comment:
IMO we can simply use `getNameInNamespace` and drop all the URL stuff. I don't think we need to support pre JRE 1.5.

Reply:
Okay, that sounds good. I removed the LDAP URL parsing and used the `getNameInNamespace` method instead.